tests/chaos/standard/test_standard.py

```diff
@@ -159,24 +159,24 @@ class TestVMInstanceTypeOperationsPodDelete:
     @pytest.mark.polarion("CNV-11108")
     @pytest.mark.first
     def test_deploy_vm(self, chaos_vms_instancetype_list, deleted_pod_by_name_prefix):
-        for vm in chaos_vms_instancetype_list:
-            vm.deploy(wait=False)
         for vm in chaos_vms_instancetype_list:
             running_vm(vm=vm)
 
     @pytest.mark.polarion("CNV-11297")
     @pytest.mark.order(after="test_deploy_vm")
     def test_restart_vm(self, chaos_vms_instancetype_list, deleted_pod_by_name_prefix):
         for vm in chaos_vms_instancetype_list:
+            if not vm.is_running():
+                vm.start(wait=True, timeout=TIMEOUT_5MIN)
```

Comment:
Maybe i'm missing something but this is not needed.
In the previous test `test_deploy_vm` you are starting it, so by saying the `not vm.is_running()` you are saying the previous test failed. so this test won't be running